frontend/src/main/scala/bloop/engine/Interpreter.scala

```diff
@@ -271,6 +275,40 @@ object Interpreter {
     state
   }
 
+  private def link(cmd: Commands.Link, state: State, sequential: Boolean): Task[State] = {
+    state.build.getProjectFor(cmd.project) match {
+      case None =>
+        Task(reportMissing(cmd.project :: Nil, state))
+      case Some(project) if project.platform != Platform.Native =>
+        Task {
+          state.logger.error("This command can only be called on a Scala Native project.")
```

Comment:
When you add Scala.js support, can we improve this message and have it be like: "Expected `$PROJECT_NAME` to be either a Scala.js or Scala Native project"?